frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -924,24 +935,50 @@ function FeatureFlagRollout({ readOnly }: FeatureFlagReadOnlyProps): JSX.Element
                                     <Col span={3}>
                                         <Field name={['multivariate', 'variants', index, 'rollout_percentage']}>
                                             {({ value, onChange }) => (
-                                                <LemonInput
-                                                    type="number"
-                                                    min={0}
-                                                    max={100}
-                                                    value={value}
-                                                    onChange={(changedValue) => {
-                                                        if (changedValue !== null && changedValue !== undefined) {
-                                                            const valueInt = parseInt(changedValue.toString())
-                                                            if (!isNaN(valueInt)) {
-                                                                onChange(valueInt)
+                                                <div>
+                                                    <LemonInput
+                                                        type="number"
+                                                        min={0}
+                                                        max={100}
+                                                        value={value}
+                                                        onChange={(changedValue) => {
+                                                            if (changedValue !== null && changedValue !== undefined) {
+                                                                const valueInt = parseInt(changedValue.toString())
+                                                                if (!isNaN(valueInt)) {
+                                                                    onChange(valueInt)
+                                                                }
                                                             }
-                                                        }
-                                                    }}
-                                                />
+                                                        }}
+                                                    />
+                                                    {_filter_groups.filter((g) => g.variant == variant.key).length >
```

Comment:
styling nit: more descriptive names for what you're filtering on please :) - and always use `===` unless there's a good reason for `==`.

```suggestion
                                                    {_filter_groups.filter((group) => group.variant == = variant.key).length >
```